doc/decisions/0001-context-in-translations.md

```diff
@@ -0,0 +1,84 @@
+  | `$.messages` | Global answer error message | |
+  | `$.sections.[*].title` | Section title | Used for section summaries, final summary and the hub |
+  | `$.sections.[*].repeat.title` | Section title (repeating section) | |
+  | `$.sections.[*].summary.items.[*].title` | Custom section summary item title | |
+  | `$.sections.[*].summary.items.[*].add_link_text` | Custom section summary list add link | |
+  | `$.sections.[*].summary.items.[*].empty_list_text` | Custom section summary empty list text | |
+  | `$..groups.[*].title` | Group title | Used as heading in summaries |
+  | `$..blocks.[*].title` | Block title | Deprecated? |
+  | `$..blocks.[*].summary.title` | List collector summary heading | Deprecated when section summary blocks are removed. |
+  | `$..blocks.[*].summary.item_title` | List collector summary item | Defines how to form the string to summarise each item in the list. Strictly speaking this doesn't need to be translated for all current usages |
+  | `$..add_block.cancel_text` | List collector add block cancel link | |
+  | `$..content.title` | Content page heading | The main heading on a Content page |
+  | `$..content.contents.[*].title` | Content page contents title | |
+  | `$..content.contents.[*].description` | Content page contents description | |
+  | `$..content.contents.[*].list` | Content page contents list | |
+  | `$..question.title` | Question text | |
+  | `$..question.description` | Question description | |
+  | `$..question.instruction` | Question instruction | Instructions for field interviewers |
+  | `$..question.definitions.[*].title` | Question definition link | |
+  | `$..question.definitions.[*].contents.[*].title` | Question definition title | |
+  | `$..question.definitions.[*].contents.[*].description` | Question definition description | |
+  | `$..question.definitions.[*].contents.[*].list` | Question definition list  | |
+  | `$..question.guidance.contents.[*].title` | Question guidance title | |
+  | `$..question.guidance.contents.[*].description` | Question guidance description | |
+  | `$..question.guidance.contents.[*].list` | Question guidance list | |
+  | `$..answers.[*].validation.messages` | Answer error message | |
+  | `$..answers.[*].label` | Answer label | |
+  | `$..answers.[*].description` | Answer description | |
+  | `$..answers.[*].playback` | Relationships playback template | Used for playback before a relationship is chosen e.g. "{person2} is {person1}'s ..." |
+  | `$..answers.[*].options.[*].label` | Answer option | |
+  | `$..answers.[*].options.[*].description` | Answer option description | |
+  | `$..answers.[*].options.[*].detail_answer.label` | Detail answer label | |
+  | `$..answers.[*].options.[*].detail_answer.description` | Detail answer description | |
+  | `$..answers.[*].options.[*].title` | Relationships answer option question text | Question text to set when this answer option is selected e.g. "Thinking about {person1}, {person2} is their husband or wife"|
```

Comment:
```suggestion
  | `$..answers.[*].options.[*].title` | Relationships answer option question text | Question text to set when an answer option is selected e.g. "Thinking about {person1}, {person2} is their husband or wife"|
```

Reply:
It's specifically this answer option, rather than any other option.